staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/etcd.go

```diff
@@ -19,24 +20,33 @@ package customresourcedefinition
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/registry/generic"
 	genericregistry "k8s.io/apiserver/pkg/registry/generic/registry"
 	"k8s.io/apiserver/pkg/registry/rest"
 	"k8s.io/apiserver/pkg/storage"
 	storageerr "k8s.io/apiserver/pkg/storage/errors"
 	"k8s.io/apiserver/pkg/util/dryrun"
+
+	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
 )
 
 // rest implements a RESTStorage for API services against etcd
 type REST struct {
 	*genericregistry.Store
 }
 
+const (
+	crdSharingPolicyAnnotation = "futurewei.k8s.io/crd-sharing-policy"
```

Comment:
not major issue - considering arktos.futurewei.com already is used somewhere else,  "arktos.futurewei.com/crd-sharing-policy" seems consistent.

Reply:
updated.